### PR TITLE
Congratulation page -> Correct style

### PR DESCRIPTION
**Pull request by Arnoldicus**
styles fixed as expected

### CODE REVIEW

**Comment by Sneaky69** [comment]
I can do that! XD?

**Comment by Nienormalny** [comment]
Could you rebase please?

**Comment by Arnoldicus** [comment]
@Nienormalny Okay, but i'm gonna make new merge request if you don't mind

File: src/components/hiscore/hiscore.component.js [hunk 1]
```diff
@@ -42,14 +44,13 @@ function HiScore(props) {
         }
     }
     return (
-        <div id="high-score-owner">
-            <h1> Congratulations!</h1>
+        <div className='congrat-container'>
+            <h1 className='congrat-text'> Congratulations!</h1>
```
Comment (lukasbrodbeck) [inline src/components/hiscore/hiscore.component.js]: Please consistently use double quotations for classNames, as in line 50

File: src/components/hiscore/hiscore.component.js [hunk 2]
```diff
@@ -4,6 +4,8 @@ import * as _                           from 'lodash';
 import {connect}                        from 'react-redux';
 import {getRandomId}                    from '../../common/widgets';
 
+
+// class HiScore extends React.Component
```
Comment (lukasbrodbeck) [inline src/components/hiscore/hiscore.component.js]: Please remove unused code

**Pull request closed without merging**